db/seeds.rb

```diff
@@ -1,128 +1,146 @@
+# rubocop:disable Rails/Output
+
+# NOTE: when adding new data, please use this class to ensure the seed tasks
+# stays idempotent.
+class Seeder
+  def initialize
+    @counter = 0
+  end
+
+  def create_if_none(klass, count = nil)
+    @counter += 1
+    plural = klass.name.pluralize
+
+    if klass.none?
+      message = ["Creating", count, plural].compact.join(" ")
+      puts "  #{@counter}. #{message}."
```

Comment:
I thought displaying the output could be nicer than using the Rails logger, so people know exactly what changed (they may miss the logs). I can change it back though if people prefer that.